Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/BocListValidationResultDispatchingValidatorTest.cs

```diff
@@ -160,6 +160,170 @@ public void DispatchValidationFailures_WithRowEditModeControlHavingInvalidValida
       Assert.That(failuresInRepository.Single().Failure.ErrorMessage, Is.EqualTo("The value contains unsupported characters: \"\u0001\" (position: 1)."));
 
       validationResultMock.Verify();
+
+      // Validates that dispatch was called to dispatching validator
+      var bocTextValue = ((Control)editableRow).FindControl("0") as BusinessObjectBoundEditableWebControl;
+      validationResultMock.Verify(_ => _.GetValidationFailures(bocTextValue.DataSource.BusinessObject, bocTextValue.Property, true));
```

Comment:
see comment in line 235

Reply:
These calles were already handled in the above Setup that uses simpleColumnDefinition.GetPRopertyPath()..., this was therefore an unrequired verify.